go/teams/service_helper.go

```diff
@@ -709,6 +733,38 @@ func AcceptSeitan(ctx context.Context, g *libkb.GlobalContext, ikey SeitanIKey)
 	return err
 }
 
+func AcceptSeitanV2(ctx context.Context, g *libkb.GlobalContext, ikey SeitanIKeyV2) error {
+	me, err := libkb.LoadMe(libkb.NewLoadUserArgWithContext(ctx, g))
```

Comment:
Don't use `LoadMe`, it's *super* slow. Use `team.currentUserUV`.  Makes sense to change the other one too (for V1)

Reply:
done